docs/error-handling.md

```diff
@@ -2,7 +2,7 @@
 
 ## Try-Catch
 
-  Using async functions means that you can try-catch `next`.
+  Using async functions mean that you can try-catch `next`.
```

Comment:
I'm not an english native speaker, but I feel it's supposed to be "means" rather than "mean" in this sentence. @niftylettuce What's your take on it?

Reply:
Either way works and people will understand

Reply:
@miwnwski I believe it should be mean rather than means as per grammar.